cmd/build.go

```diff
@@ -153,7 +153,12 @@ func newBuildCmd() *cobra.Command {
 				srcPath = scm.LocalPath()
 			}
 
-			config, err := configNew(filepath.Join(sdlocalDir, "config"))
+			configList, err := newConfigList(filepath.Join(sdlocalDir, "config"))
+			if err != nil {
+				return err
+			}
+
+			config, err := configList.Get(configList.Current)
```

Comment:
I feel it is better to configList have `Current` function.

Reply:
I think `Config.Current` method conflicts with `Config.Current` field.

Reply:
I see. How about to create `Config.CurrentEntry`?

Reply:
What motivates you to create `Config.CurrentEntry` function?
Are there anything to make it merit?

Reply:
It's just that I felt happy to take the current settings directly.